ochap-backend/src/main/resources/templates/productNotFound.html

```diff
@@ -0,0 +1,348 @@
+                                        <li class="menu_iconb">
+                                            <a href="#">Log in <img style="margin-right: 15px;" src="/icon/5.png" alt="#" /> </a>
+                                        </li>
+                                        <li class="menu_iconb">
+                                            <a href="#">Signup<img style="margin-left: 15px;" src="/icon/6.png" alt="#" /></a>
+                                        </li>
+                                        <li class="tytyu">
+                                            <a href="#"> <img style="margin-right: 15px;" src="/icon/2.png" alt="#" /></a>
+                                        </li>
+                                        <li class="menu_iconb">
+                                            <a href="#"><img style="margin-right: 15px;" src="/icon/3.png" alt="#" /></a>
+                                        </li>
+
+                                        <li>
+                                            <button type="button" id="sidebarCollapse">
+                                                <img src="/images/menu_icon.png" alt="#" />
+                                            </button>
+                                        </li>
+                                    </ul>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <!-- end header inner -->
+
+                <!-- end header -->
+            </div>
+        </header>
+
+        <style>
+            .product {
+                column-gap: 16px;
+                display: grid;
+                grid-template-columns: auto 1fr;
+                grid-template-rows: 32px 32px 1fr 1fr;
+                margin: 16px;
+                row-gap: 16px;
+            }
+            .product img {
+                grid-column: 1;
+                grid-row: 1 / span 3;
+            }
+            .product h1 {
+                grid-column: 2;
+                grid-row: 1;
+            }
+            .product .product-price {
+                align-self: center;
+                grid-column: 2;
+                grid-row: 2;
+            }
+            .product form {
+                grid-column: 2;
+                grid-row: 3;
+            }
+            .product .product-add-to-shopping-cart {
+                background: #ee3e0f;
+                color: #fff;
+                cursor: pointer;
+                padding: 7px 26px;
+            }
+            .product .product-add-to-shopping-cart:hover {
+                background: #fff;
+                border-radius: 20px;
+                color: #000;
+                transition: all 0.4s ease-in;
+            }
+            .product .product-description {
+                grid-column: 1 / span 2;
+                grid-row: 4;
+            }
+        </style>
+        <main>
```

Comment:
This file looks very much like `product.html`. Why don't we merge them both and display the adequate text as a function of what is available?

Something like
```html
<div th:if="${product ne null}">
   <!-- Show the product -->
</div>
<div th:if="${product == null}">
            <h1>The product you are looking for could not be found</h1>
</div>
```

See https://attacomsian.com/blog/thymeleaf-handling-null-values